CYLGame/www/templates/grid.html

```diff
@@ -136,11 +136,17 @@
             });
         }
         function testCode(opponents) {
+            let token_value;
+            if (!localStorage.user_token) {
+                token_value = "";
+            } else {
+                token_value = localStorage.user_token;
+            }
             $("#debugTable").html("");
             $.ajax({
                 type: "POST",
                 url: $SCRIPT_ROOT + 'sim',
-                data: JSON.stringify({code: editor.getValue(), seed: window.seed, opponents: opponents}),
+                data: JSON.stringify({code: editor.getValue(), token:token_value, seed: window.seed, opponents: opponents}),
```

Comment:
Nit: Do you want to add a space after the colon here?

Reply:
Yes I do :)